packages/resilience/src/middleware.ts

```diff
@@ -75,8 +75,8 @@ export type Middleware<
   ReturnT,
   ContextT extends MiddlewareContext<ArgumentT>
 > = (
-  options: MiddlewareIn<ArgumentT, ReturnT, ContextT>
-) => MiddlewareOut<ArgumentT, ReturnT>;
+  options: MiddlewareOptions<ArgumentT, ReturnT, ContextT>
```

Comment:
[req] Update the jsdoc. It still mentions `MiddlewareIn`

Reply:
see here: https://github.com/SAP/cloud-sdk/pull/1234

Reply:
No, I meant the comment in line 62. 
 

> The input is the MiddlewareIn containing the initial function and some context information e.g. axios request and the request context.